cmd/pcr0tool/commands/dumpregisters/command.go

```diff
@@ -64,7 +55,16 @@ func (cmd Command) Execute(args []string) {
 	}
 }
 
-func printRegister(reg registers.Register) {
+// PrintRegisters outputs registers in a detailed human-readable format
+func PrintRegisters(regs registers.Registers) {
+	for _, reg := range regs {
+		fmt.Printf("\n")
+		PrintRegister(reg)
+	}
+}
+
+// PrintRegister outputs a single register in a detailed human-readable format
+func PrintRegister(reg registers.Register) {
```

Comment:
If we want to publish them, then it is required to move it to [`cmd/pcr0tool/commands/dumpregisters/helpers`](https://github.com/9elements/converged-security-suite/tree/master/cmd/pcr0tool/commands/dumpregisters/helpers). Otherwise there's too much dependencies for too lightweight logic.